soc/arm/nuvoton_npcx/common/soc_dt.h

```diff
@@ -542,4 +542,137 @@
 	}
 
+/**
+ * @brief Get a node from path '/vsby-psl-in-list' which has a property
+ *        'psl-in-pads' contains Power Switch Logic (PSL) input pads which are
+ *        in charge of detecting wake-up events on VSBY power domain.
+ *
+ * @return node identifier with that path.
+ */
+#define NPCX_DT_NODE_PSL_IN_LIST  DT_PATH(vsby_psl_in_list)
+
+/**
+ * @brief Length of npcx_psl_in structures in 'psl-in-pads' property
+ *
+ * @return length of 'psl-in-pads' prop which type is 'phandles'
+ */
+#define NPCX_DT_PSL_IN_ITEMS_LEN DT_PROP_LEN(NPCX_DT_NODE_PSL_IN_LIST, \
+								psl_in_pads)
+
+/**
+ * @brief Get phandle from 'psl-in-pads' prop which type is 'phandles' at index
+ *        'i'
+ *
+ * @param i index of 'psl-in-pads' prop which type is 'phandles'
+ * @return phandle from 'psl-in-pads' prop at index 'i'
+ */
+#define NPCX_DT_PHANDLE_FROM_PSL_IN_NODE(i) \
+	DT_PHANDLE_BY_IDX(NPCX_DT_NODE_PSL_IN_LIST, psl_in_pads, i)
+
+/**
+ * @brief Get phandle from 'pinctrl-0' prop which type is 'phandles' at index
+ *        'i'
+ *
+ * @param i index of 'psl-in-pads' prop which type is 'phandles'
+ * @return phandle from 'pinctrl-0' prop at index 'i'
+ */
+#define NPCX_DT_PHANDLE_FROM_PSL_PINMUX_NODE(i) \
+	DT_PHANDLE(NPCX_DT_PHANDLE_FROM_PSL_IN_NODE(i), pinctrl_0)
+
+/**
+ * @brief Get phandle from 'polarity-0' prop which type is 'phandles' at index
+ *        'i'
+ *
+ * @param i index of 'psl-in-pads' prop which type is 'phandles'
+ * @return phandle from 'polarity-0' prop at index 'i'
+ */
+#define NPCX_DT_PHANDLE_FROM_PSL_POLARITY_NODE(i) \
+	DT_PHANDLE(NPCX_DT_PHANDLE_FROM_PSL_IN_NODE(i), polarity_0)
+
+/**
+ * @brief Construct a npcx_alt structure from 'pinctrl-0' property at index 'i'
+ *        of 'psl-in-pads' prop.
+ *
+ * @param i index of 'psl-in-pads' prop which type is 'phandles'
+ * @return npcx_alt item from 'pinctrl-0' property at index 'i'
+ */
+#define NPCX_DT_PSL_IN_ALT_CONF_BY_IDX(i)                                         \
+	{                                                                         \
+	  .group = DT_PHA(NPCX_DT_PHANDLE_FROM_PSL_PINMUX_NODE(i), alts, group),  \
+	  .bit = DT_PHA(NPCX_DT_PHANDLE_FROM_PSL_PINMUX_NODE(i), alts, bit),      \
+	  .inverted = DT_PHA(NPCX_DT_PHANDLE_FROM_PSL_PINMUX_NODE(i), alts, inv), \
+	},
+
+/**
+ * @brief Construct a npcx_alt structure from 'polarity-0' property at index 'i'
+ *        of 'psl-in-pads' prop.
+ *
+ * @param i index of 'psl-in-pads' prop which type is 'phandles'
+ * @return npcx_alt item from 'pinctrl-0' property at index 'i'
+ */
+#define NPCX_DT_PSL_IN_POL_CONF_BY_IDX(i)                                           \
+	{                                                                           \
+	  .group = DT_PHA(NPCX_DT_PHANDLE_FROM_PSL_POLARITY_NODE(i), alts, group),  \
+	  .bit = DT_PHA(NPCX_DT_PHANDLE_FROM_PSL_POLARITY_NODE(i), alts, bit),      \
+	  .inverted = DT_PHA(NPCX_DT_PHANDLE_FROM_PSL_POLARITY_NODE(i), alts, inv), \
```

Comment:
the indentation on these lines (as well as macro above) looks rather funky... normally use tabs to indent struct members instead of just two spaces

Reply:
Done.